supervisord.conf

```diff
@@ -66,7 +66,7 @@ stopasgroup = true
 [program:prometheus]
 priority = 7
 user = pmm
-command = /usr/sbin/prometheus -config.file=/etc/prometheus.yml -storage.local.path=/opt/prometheus/data -web.listen-address=:9090 -storage.local.retention=ENV_METRICS_RETENTION -storage.local.memory-chunks=ENV_METRICS_MEMORY -storage.local.chunk-encoding-version=2 -web.console.libraries=/usr/share/prometheus/console_libraries -web.console.templates=/usr/share/prometheus/consoles -web.external-url=http://localhost:9090/prometheus/
+command = /usr/sbin/prometheus -config.file=/etc/prometheus.yml -storage.local.path=/opt/prometheus/data -web.listen-address=:9090 -storage.local.retention=ENV_METRICS_RETENTION --storage.local.target-heap-size=ENV_METRICS_MEMORY_MULTIPLIED -storage.local.chunk-encoding-version=2 -web.console.libraries=/usr/share/prometheus/console_libraries -web.console.templates=/usr/share/prometheus/consoles -web.external-url=http://localhost:9090/prometheus/
```

Comment:
@AlekSi, let me explain, in this post you can see:
```
command=/bin/prog
        -arg1 value
        -arg2 value
```
will be split to `['/bin/prog', '-arg1', 'value', '-arg2', 'value']`
it is working fine if you have simple command and arguments, but we have overcomplicated commands like `bash -c "sleep 7 && /usr/sbin/orchestrator http"`
in this command we cannot split 3rd argument becase command become broken
for example
``` 
command = bash -c "
    sleep 7 \
    && /usr/sbin/orchestrator http
"
```
will be splited incorectly.